dotCMS/src/integration-test/java/com/dotmarketing/cms/urlmap/URLMapAPIImplTest.java

```diff
@@ -103,8 +99,9 @@ public void shouldReturnNullWhenTheContentNotExists()
 
     @Test
     public void shouldNotMatchUrlStaringWithAPI()
```

Comment:
what is the purpose of this test? Maybe renaming the test makes it clear

Reply:
or a javadoc with the description